SwiftShell/PrintWorkingDirectory.swift

```diff
@@ -8,11 +8,19 @@
 
 import Foundation
 
-public func pwd() -> String {
+public func pwd(folding: Bool = false) -> String {
     print(command: "pwd")
     var buffer: [Int8] = Array(repeating: 0, count: Int(PATH_MAX))
     getcwd(&buffer, buffer.count)
-    let output = String(cString: buffer) ?? ""
+    var output = String(cString: buffer) ?? ""
+    
+    let outputArr = output.characters.split{$0 == "/"}.map(String.init)
+
+    if(output.characters.count > 15 && folding == true) {
+        output = outputArr.first! + "/.../" + outputArr.last!
```

Comment:
got an error using "" 

Cannot convert value of type 'String' to expected argument type '@noescape (SubSequence) throws -> _'